p22/src/unit_tests.rs

```diff
@@ -0,0 +1,32 @@
+#[cfg(test)]
+
+mod tests {
+    #[test]
+    fn tests_celsius2farenheit() {
```

Comment:
Unit tests usually reside in the same file as the testing functions, i.e. it's better to move this to `calc.rs`.